libs/sdk-ui-charts/src/highcharts/adapter/plugins/3rdParty/grouped-categories.ts

```diff
@@ -1,3 +1,5 @@
+// (C) 2021 GoodData Corporation
+
```

Comment:
not a laywer, but unsure whether this copyright is ok here.

Reply:
you are right, removed